www/conf/nginx.conf

```diff
@@ -34,9 +34,6 @@ http {
     # if rate limits are exceeded, clients will receive a 503 answer (try again later)
     # rate limits are exceeded when the 'leaky bucket' is full
 
-    # set up one bucket per remote ip for general purpose
-    limit_req_zone $binary_remote_addr zone=perip-general:100m rate=30r/s;
```

Comment:
it appears that this bucket was not used anyways, at least `perip-general` does not appear anywhere in current nginx configuration